### DISCUSSION TITLE
Where settings which are accessed trough the left sidebar are stored ?

### DISCUSSION THREAD

**Discussion by ilias-7565** [termux/termux-app#3834] [Q&A] [answered]
.

**Comment by AllComputerWorks4U** [comment, marked as answer]
# Termux settings file:
Termux settings file should be stored on this directory:
`/data/data/com.termux/files/home/.termux/termux.properties`
# Edit Termux settings file:
If you are looking for edit the settings of termux you can find the settings file on the .termux directory running the command:
`cd ./.termux/`
Here is the file called `termux.properties`.
# After editing properties file:
Run the following command on termux:
`termux-reload-settings`.
# Source:
[Termux Official Wiki](https://wiki.termux.com/wiki/Terminal_Settings)
Hope I helped you. Bye
